applications.json

```diff
@@ -1006,7 +1006,9 @@
       "repo_url" : "https://github.com/vineetchoudhary/AppBox-iOSAppsWirelessInstallation",
       "title" : "AppBox",
       "screenshots" : [
-
+        "https://github.com/getappbox/Home/raw/master/Images/UploadIPA-Dark.png?raw=true",
```

Comment:
Please remove ?raw=true, it cause redirection and ci mark it as an issue.

Reply:
Done. Please recheck.